rudder-agent/SOURCES/patches/cfengine/15636-ignore-symlink-check-on-lxc.patch

```diff
@@ -0,0 +1,142 @@
+--- cfengine-source.orig/libenv/unix_iface.c	2022-11-09 17:33:16.000000000 +0100
++++ cfengine-source/libenv/unix_iface.c	2023-04-19 11:15:16.758412820 +0200
+@@ -1346,12 +1346,17 @@
+ void GetNetworkingInfo(EvalContext *ctx)
+ {
+     const char *procdir_root = GetRelocatedProcdirRoot();
++    int promiser_pid = (int) getpid();
+ 
+     Buffer *pbuf = BufferNew();
+ 
+     JsonElement *inet = JsonObjectCreate(2);
+ 
+-    BufferPrintf(pbuf, "%s/proc/net/netstat", procdir_root);
++    BufferPrintf(pbuf, "%s/proc/%d/net/netstat", procdir_root, promiser_pid);
++    if (!FileCanOpen(BufferData(pbuf), "r"))
++    {
++        BufferPrintf(pbuf, "%s/proc/net/netstat", procdir_root);
```

Comment:
shouldn't you change include the promiser_pid in this out ?

Reply:
???